src/abstractitensornetwork.jl

```diff
@@ -186,7 +243,7 @@ const map_inds_label_functions = [
   :setprime,
   :noprime,
   :replaceprime,
-  :swapprime,
+  :swapprime, # TODO: fix this one (broken)
```

Comment:
Would it be possible to fix this case if it's broken? If it's something subtle, could you add a `@test_broken` style unit test (https://docs.julialang.org/en/v1/stdlib/Test/#Broken-Tests) to flag this as needing fixing?

Reply:
I haven't really thought about what is going wrong for this specific case, I just noticed it in passing and added a reminder. It might not be a straightforward fix with the current implementation, but I would have to actually look at it. There's no tests for the priming and tagging part yet anyway, I can sort this out when I get to that.

Reply:
Yeah, ideally we would comment it out and then add `@test_broken` as a reminder for us to investigate it.

Reply:
Done.